packages/contracts-bedrock/contracts/test/SequencerFeeVault.t.sol

```diff
@@ -73,4 +88,33 @@ contract SequencerFeeVault_Test is Bridge_Initializer {
         // The withdrawal was successful
         assertEq(vault.totalProcessed(), amount);
     }
+
+    function test_withdraw_toL2_succeeds() external {
```

Comment:
This feels like it should in another test contract with the asymmetry of the `vm.etch` happening in `setUp` vs a test fn